docker/images/pinot-base/README.md

```diff
@@ -23,42 +23,32 @@
 
 This is the base docker image to build [Apache Pinot](https://github.com/apache/pinot).
 
-## How to build a docker image
+## Build and publish the docker image
 
-Arguments:
+Here is
+the [Github Action task](https://github.com/apachepinot/pinot-fork/actions/workflows/build-pinot-docker-base-image.yml)
+to build and publish pinot base docker images.
 
-`JAVA_VERSION`: The Java Build and Runtime image version. Default is `11`
+This task can be triggered manually to build the cross platform(amd64 and arm64v8) base image.
 
-`OPENJDK_IMAGE`: Base image to use for Pinot build and runtime, e.g. `arm64v8/openjdk`. Default is `openjdk`.
+The build shell is:
 
-Usage:
+For Amazon Corretto 11:
 
 ```SHELL
-docker build -t apachepinot/pinot-base-build:openjdk11 --no-cache --network=host --build-arg JAVA_VERSION=11 -f pinot-base-build/Dockerfile .
+docker buildx build --no-cache --platform=linux/arm64,linux/amd64 --file pinot-base-build/amazoncorretto.dockerfile --tag apachepinot/pinot-base-build:11-amazoncorretto --push .
 ```
 
 ```SHELL
-docker build -t apachepinot/pinot-base-runtime:openjdk11 --no-cache --network=host --build-arg JAVA_VERSION=11 -f pinot-base-runtime/Dockerfile .
+docker buildx build --no-cache --platform=linux/arm64,linux/amd64 --file pinot-base-runtime/amazoncorretto.dockerfile --tag apachepinot/pinot-base-runtime:11-amazoncorretto --push .
 ```
 
-Note that if you are not on arm64 machine, you can still build the image by turning on the experimental feature of docker, and add `--platform linux/arm64` into the `docker build ...` script, e.g.
-
-```SHELL
-docker build -t apachepinot/pinot-base-build:openjdk11-arm64v8 --platform linux/arm64 --no-cache --network=host --build-arg JAVA_VERSION=11 --build-arg OPENJDK_IMAGE=arm64v8/openjdk -f pinot-base-build/Dockerfile .
-```
+For MS OpenJDK, the build shell is:
 
 ```SHELL
-docker build -t apachepinot/pinot-base-runtime:openjdk11-arm64v8 --platform linux/arm64 --no-cache --network=host --build-arg JAVA_VERSION=11 --build-arg OPENJDK_IMAGE=arm64v8/openjdk -f pinot-base-runtime/Dockerfile .
+docker buildx build --no-cache --platform=linux/arm64,linux/amd64 --file pinot-base-build/amazoncorretto.dockerfile --tag apachepinot/pinot-base-build:11-ms-openjdk --push .
```

Comment:
Is this a typo? I would expect that for MS OpenJDK we should use `ms-openjdk.dockerfile` as base image.